device_policy.py

```diff
@@ -44,59 +64,46 @@ def serialize_policy(self, private_key):
 
   def import_policy(self, policy_dict):
     proto_map_path = schema_dir / "device_policy_proto_map.yaml"
-    legacy_map_path = schema_dir / "legacy_device_policy_proto_map.yaml"
     proto_map = yaml.safe_load(proto_map_path.read_bytes())
-    legacy_proto_map = yaml.safe_load(legacy_map_path.read_bytes())
 
-    #get list of device policy keys
-    for key, item in legacy_proto_map.items():
-      if not key or not item:
-        continue
-      proto_map[key] = item
     valid_names = list(proto_map.keys())
 
-    #find which names in the policy dict are device policies
+    # Find which names in the policy dict are device policies.
     dict_names = list(policy_dict.keys())
     imported_names = list(set(valid_names).intersection(dict_names))
 
-    #iterate through the policies and set the values in the device settings
-    message_dict = {}
+    # Iterate through the policies and set the values in the device settings.
     for policy_name in imported_names:
       policy_key = proto_map[policy_name]
-      if type(policy_key) == list:
-        print(f"skipped policy {policy_name} since multiple mappings are not supported yet")
-        continue
       
-      policy_value = policy_dict[policy_name]["value"]
+      policy_value = policy_dict[policy_name]
       try: 
         self.set_policy_item(policy_key, policy_value)
       except ParseError:
         print(f"skipped policy {policy_name} due to invalid parameters")
-    
-    print(self.device_settings)
 
   def set_policy_item(self, policy_key, policy_value):
-    #see if we need to fix a non-standard mapping of values
+    # See if we need to fix a non-standard mapping of values.
     policy_converted = self.fix_policy_value(policy_key, policy_value)
     
-    #construct a dict for the proto message
+    # Construct a dict for the proto message.
     key1, key2 = policy_key.split(".")
     policy_json = {
       key1: {
         key2: policy_converted
       }
     }
-    #attempt to parse and insert it into the device settings
+    # Attempt to parse and insert it into the device settings.
     ParseDict(policy_json, self.device_settings)
   
-  #sometimes policy json values dont match with the protobuf values
+  # Sometimes policy json values dont match with the protobuf values.
   def fix_policy_value(self, policy_key, policy_value):
-    if policy_key == "device_local_accounts.account":
+    if policy_key == "DeviceLocalAccounts":
```

Comment:
Why has this been changed?

Reply:
I'm unsure, I think I just did something dumb. Doing this actually worked, though, because in the new format nothing had to be fixed, so the fixer not working was actually a good thing.